data/human/hails.txt

```diff
@@ -1670,6 +1670,47 @@ phrase "friendly civilian"
 		"The next event will be great!"
 		"That's real entertainment!"
 
+phrase "friendly civilian"
+	word
+		"I"
+		"Me and my friend"
+	word
+		" "
+	word
+		"had a job"
+		"used to spend time on"
```

Comment:
```suggestion
		"used to spend time"
```
Another thing: for some of the "word"s below, having `on` at the end of this "word" is appropriate, and sometimes it isn't:
`used to spend time on farming in the dirt belt`
I'd recommend removing it here and adding `on` to the beginning of later "word"s where it's appropriate.